### ISSUE TITLE
Training bolcked

### ISSUE THREAD

**Issue by charlesCXK** [ycszen/TorchSeg#59]
When I train my network, the program was blocked after the first epoch. I don't know why this happeded.

```
Epoch0/800 Iter20/20: lr=2.00e-02 loss=2.75: [00:44<00:00, 1.32it/s]
Epoch0/800 Iter20/20: lr=2.00e-02 loss=2.75: [00:44<00:00, 1.31it/s]
[00:00<?,?it/s]
```

**Comment by charlesCXK** [comment]
I have two machines, the code can run successfully on the first machine, but it was blocked on the second.

**Comment by zhangyunming** [comment]
i have meet the same.


10 14:37:50 PyTorch Version 1.0.1, Furnace Version 0.1.1
10 14:37:50 PyTorch Version 1.0.1, Furnace Version 0.1.1
Epoch0/140 Iter200/200: lr=9.94e-03 loss=1.87: [00:52<00:00,10.18it/s]

[00:00<?,?it/s]10 14:38:47 Saving checkpoint to file /disk3t-2/zym/TorchSeg/log/cityscapes.bisenet.X39/snapshot/epoch-0.pth
10 14:38:47 Save checkpoint to file /disk3t-2/zym/TorchSeg/log/cityscapes.bisenet.X39/snapshot/epoch-0.pth, Time usage:
	prepare snapshot: 0.00449681282043457, IO: 0.08965277671813965
[00:00<?,?it/s]

**Comment by charlesCXK** [comment]
Set **num_workers=0** in the dataloader works, but it is too slow... Is there any other solutions?

**Comment by zhangyunming** [comment]
No. i have no solutions, as you said ,i set num_workers=0 in the dataloader.py and set
 """Display Config"""
C.snapshot_iter = 1 in config.py 

but it will take 400 hours to finish the trainning.  cry~~

**Comment by ycszen** [comment]
@charlesCXK Maybe there are some problems on the second machine?

**Comment by ycszen** [comment]
@zhangyunming The snapshot_iter just affects the model-saving.

If you make it successful by setting num_workers=0, maybe you need to enlarge the share-memory, I guess.

**Comment by charlesCXK** [comment]
> @charlesCXK Maybe there are some problems on the second machine?

I don't know... The second machine is company's cluster.

**Comment by ycszen** [comment]
@charlesCXK Maybe you need to enlarge the share-memory of the second machine on the cluster.

**Comment by charlesCXK** [comment]
> @charlesCXK Maybe you need to enlarge the share-memory of the second machine on the cluster.

Thanks! I asked to a colleague in the IT department but he said that he  could not enlarge it. 2333
